siff-dr-router/router-netfilter.go

```diff
@@ -1,20 +1,73 @@
 package main
 
 import (
+	"bytes"
 	"log"
 
 	"github.com/ThomasJClark/cs4516project/pkg/go-netfilter-queue"
+	"github.com/google/gopacket"
+	"github.com/google/gopacket/layers"
 )
 
+func Serialize(ipLayer *layers.IPv4) ([]byte, error) {
```

Comment:
This looks like a repeat of the earlier function, can we refactor to only have one version?